### PR TITLE
feat(doctor): add doctor command

### PR DESCRIPTION
**Pull request by smothiki**

### CODE REVIEW

File: parser/doctor.go [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+Usage: deis doctor:send [options]
+
+`
+
+	args, err := docopt.Parse(usage, argv, true, "", false, true)
+	if err != nil {
+		return err
+	}
+
+	// The 1000 is fake for now until API understands limits
```
Comment (Joshua-Anderson) [inline parser/doctor.go]: I don't understand this comment... is it still relevant?

File: cmd/doctor.go [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/deis/controller-sdk-go/doctor"
+	"github.com/deis/workflow-cli/settings"
+)
+
+// PsScale scales an app's processes.
```
Comment (Joshua-Anderson) [inline cmd/doctor.go]: Copy-Paste Comment Errror

**Comment by smothiki** [comment]
requires https://github.com/deis/controller-sdk-go/pull/12

**Comment by smothiki** [comment]
closing this as this is not in the scope

**Pull request closed without merging**